cg/store/models.py

```diff
@@ -1116,6 +1116,15 @@ class PacBioSequencingRun(InstrumentRun):
     failed_reads: Mapped[BigInt]
     failed_yield: Mapped[BigInt]
     failed_mean_read_length: Mapped[BigInt]
+    barcoded_hifi_reads: Mapped[BigInt | None]
+    barcoded_hifi_reads_percentage: Mapped[Num_6_2 | None]
+    barcoded_hifi_yield: Mapped[BigInt | None]
+    barcoded_hifi_yield_percentage: Mapped[Num_6_2 | None]
+    barcoded_hifi_mean_read_length: Mapped[BigInt | None]
+    unbarcoded_hifi_reads: Mapped[BigInt | None]
+    unbarcoded_hifi_reads_percentage: Mapped[Num_6_2 | None]
+    unbarcoded_hifi_yield: Mapped[BigInt | None]
+    unbarcoded_hifi_mean_read_length: Mapped[BigInt | None]
 
```

Comment:
Do this change when we have the barcode metrics parsed and included to the database
```suggestion
    barcoded_hifi_reads: Mapped[BigInt]
    barcoded_hifi_reads_percentage: Mapped[Num_6_2]
    barcoded_hifi_yield: Mapped[BigInt]
    barcoded_hifi_yield_percentage: Mapped[Num_6_2]
    barcoded_hifi_mean_read_length: Mapped[BigInt]
    unbarcoded_hifi_reads: Mapped[BigInt]
    unbarcoded_hifi_reads_percentage: Mapped[Num_6_2]
    unbarcoded_hifi_yield: Mapped[BigInt]
    unbarcoded_hifi_mean_read_length: Mapped[BigInt]

```

Reply:
fixed